_ext/pipeline.rb

```diff
@@ -21,12 +21,13 @@
 require 'disqus_more'
 require 'lower_case_paths'
 require 'customer_portal'
+require 'events'
 require 'connectors'
 require 'vault'
 require 'aweplug/helpers/define'
 
 Awestruct::Extensions::Pipeline.new do
-  
```

Comment:
Lots ow WS changes in this file. Take a look at the diff, you'll see how difficult it is to find your actual change.